paper-dropdown-menu.html

```diff
@@ -110,16 +110,17 @@
     }
 
     paper-ripple {
-      top: 20px;
```

Comment:
I think the `top` here was meant to accomodate the positioning of the floating label in the input. Does this still look okay when there is a floating label?